content/release-notes/1.51.0.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-09-03T17:00:00-00:00
+linktitle: "1.51.0"
+title: "1.51.0"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<features>}}
+{{</features>}}
+
+{{<changes>}}
+{{</changes>}}
+
+{{<fixes>}}
```

Comment:
For now, we can follow the existing style convention. But it's redundant under the Bug Fixes category to begin every sentence with "Fixed a bug....". From a UX perspective, that is well defined by the section title. Unless there are objections, I would suggest going straight into meat of each bullet point moving forward. So for this particular RN, if I were writing it, would go....

{{<fixes>}}

- KOTS was failing to detect private images if the domains were blocked by a firewall during online installs.
- The set config command failed when the application config had a required password value that was already set.
- The LocalImageName function did not require private images when `proxyPublicImages` is set.
- The secret used to pull private images was removed during snapshot restore when multiple applications are installed.
- When using an API, the Helm render chart was not supported by the target Kubernetes cluster.
- Newer style charts with private images were failing to kustomize. *See line item question below*
- Newer style charts with private images were coexisting with legacy style charts causing failed deployments. 

{{</fixes>}}